plugin/src/main/java/io/jenkins/plugins/coverage/metrics/restapi/CoverageApi.java

```diff
@@ -1,4 +1,4 @@
-package io.jenkins.plugins.coverage.metrics.steps;
+package io.jenkins.plugins.coverage.metrics.restapi;
```

Comment:
You need to add that new package to https://github.com/jenkinsci/code-coverage-api-plugin/blob/master/plugin/src/test/resources/design.puml to get the architecture tests running

Reply:
Done.